### DISCUSSION TITLE
Masked credit cards

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1479] [Feature Requests]
Hello!
Please add the "masked cards" technology similar to the one implemented in the Blur service from Abine.

Here are the details of what I mean:
https://www.abine.com/blog/2018/how-to-create-a-virtual-credit-card-part1/

This would be very useful for those services where you should buy, but you don't trust them. Also, it would be ideal for hiding identities, because marketers would no longer be able to track users based on their card data

**Comment by nguyenkims** [comment]
We have this feature request tracked in https://github.com/simple-login/app/discussions/1278
